config/libopensplice69_bionic.yaml

```
@@ -0,0 +1,6 @@
+name: libopensplice69_bionic
+method: http://packages.osrfoundation.org/gazebo/ubuntu-prerelease
+suites: [bionic]
```

```suggestion
suites: [bionic, xenial]
```

Combining Xenial and Bionic into a single PR.